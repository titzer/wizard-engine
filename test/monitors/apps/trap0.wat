(module
  (func $start (export "_start")
    (nop)
    (nop)
    (call $main)
    (drop)
  )
  (func $main (export "main") (result i32)
    (nop)
    (nop)
    (nop)
    (nop)
    (nop)
    (call $foo)
    (i32.const 42)
  )
  (func $foo
    (nop)
    (nop)
    (nop)
    (unreachable)
  )
)
