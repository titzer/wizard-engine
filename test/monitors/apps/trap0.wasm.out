<wasm func "main"> +6
  <wasm func "foo"> +4
    !trap[UNREACHABLE]
